incubator/virtualcluster/pkg/syncer/syncer.go

```diff
@@ -52,12 +57,13 @@ func New(
 ) *Syncer {
 	syncer := &Syncer{
 		secretClient: secretClient,
+		stopChan:     signals.SetupSignalHandler(),
 	}
 
 	// Handle VirtualCluster add&delete
 	virtualClusterInformer.Informer().AddEventHandler(
 		cache.ResourceEventHandlerFuncs{
-			AddFunc:    syncer.onVirtualClusterAdd,
+			UpdateFunc: syncer.onVirtualClusterUpdate,
```

Comment:
AddFunc is required to load the existing vc

Reply:
I will added back.